app/src/main/java/net/skyscanner/backpack/demo/MainActivity.kt

```diff
@@ -41,21 +64,90 @@ class MainActivity : BpkBaseActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_component_list)
+    setContent {
+      BpkTheme {
+        Column {
```

Comment:
It might be a little cleaner to extract the layout to a separate function as well

Reply:
Thanks! Do you mean extract everything that's inside that column?

Reply:
Including the column, yes :)

Reply:
Done now, thanks!